content/smart.md

```diff
@@ -54,7 +54,35 @@ The recommended screen resolution varies based on the available screen's real es
 
 ### Browser Requirement ###
 
-Apps should be designed to work in any browser for best user experience. For provider facing apps that run on the *Cerner Millennium* platform, Cerner currently supports only the embedded Internet Explorer browser (IE) at the moment. The minimum-supported embedded browser is IE 10. The latest supported browser varies based on the version of the browser that is currently installed on the device.
+Apps should be designed to work in any browser for best user experience. For provider facing apps that run on the *Cerner Millennium* platform, Cerner currently supports only the embedded Internet Explorer browser (IE) at the moment. The embedded browser control that we use is [IWebBrowser2 C++ interface](https://msdn.microsoft.com/en-us/library/aa752127(v=vs.85).aspx) by Microsoft. The minimum-supported embedded browser is IE 10. Apps *must* be designed to work with IE 10. The latest supported browser varies based on the version of the browser that is currently installed on the device.
```

Comment:
You could mention that Microsoft offers a free IE 10 VM for application testing prior to embedding. https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/